sgxs-loaders/src/isgx/mod.rs

```diff
@@ -207,10 +310,27 @@ impl EnclaveLoad for InnerDevice {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum DriverVersion {
+    External,
+    Upstream,
```

Comment:
I don't think Upstream is the best name for this.
Today it may be upstream but it might not be tomorrow

Reply:
Any suggestions?

Reply:
I remember that I was confused about what Upstream means.
Does it mean the driver that is in the master branch in the linux repo? In that case the name Upstream is fine

If it's a specific version of the driver then the name should reflect that. 

But I think my concerns come from not understanding the context. Maybe a comment would help understand it

Reply:
Yes upstream here means part of the Linux kernel